Jenkinsfile

```diff
@@ -0,0 +1,129 @@
+@Library('pipeline-library')
+import com.genesys.jenkins.Service
+
+def notifications = null
+
+def isReleaseBranch() {
+    return env.SHORT_BRANCH.equals('main');
+}
+
+pipeline {
+  agent { label 'dev_mesos_v2' }
+  options {
+    quietPeriod(480)
+    disableConcurrentBuilds()
+  }
+
+  environment {
+    NPM_UTIL_PATH = "npm-utils"
+    REPO_DIR = "repo"
+    SHORT_BRANCH = env.GIT_BRANCH.replaceFirst(/^origin\//, '');
+    NPM_TOKEN = credentials('2844c47b-19b8-4c5f-b901-190de49c0883')
```

Comment:
Can it be exposed? It should be a parameter we set on Jenkins environment while running this.

Reply:
I can expose the credentials id as a parameter, yes. I can't expose the credentials themselves.